pinot-query-planner/src/test/resources/queries/WindowFunctionPlans.json

```diff
@@ -1278,6 +1278,54 @@
           "\n"
         ]
       },
+      {
+        "description": "single OVER(ORDER BY) row_number and select col with global order by on different column as inside over",
```

Comment:
What about corresponding runtime tests for these variants ? Especially for the JOIN one ?

Can be done separately if not already covered